src/util.ts

```diff
@@ -1088,14 +1088,18 @@ export function RGBToHex(red: number, green: number, blue: number): string {
 export interface ColorObject {
   background?: string
   border?: string
-  hover?: {
-    border?: string
-    background?: string
-  }
-  highlight?: {
-    border?: string
-    background?: string
-  }
+  hover?:
```

Comment:
I think I don't understand this ts syntax. @Thomaash Can you give me a hint!?

Reply:
- `?` means that it may or may not be present
- `|` lists alternatives (in this case a string or an object with optional border and background string properties)

Reply:
Ha I see: Its just the type definition! Thanks!